Screen Wake Lock API subedit (#31208)
mdn/content: files/en-us/web/api/wakelocksentinel/released/index.md

{{APIRef("Screen Wake Lock API")}}{{SecureContext_Header}}

The **`released`** read-only property of the {{domxref("WakeLockSentinel")}} interface returns a boolean that indicates whether a {{domxref("WakeLockSentinel")}} has been released.

> The `WakeLockSentinel` is released when the associated platform screen wake lock is revoked, and then remains `true`.

Reviewer: ```suggestion
The `WakeLockSentinel` is released when the associated platform screen wake lock is revoked; `released` will return `true` thereafter.
```

Author: Better. Slight mod, because I find the term "thereafter" a little clunky.
```suggestion
The `WakeLockSentinel` is released when the associated platform screen wake lock is revoked; afterwards `released` will always return `true`.
```

Reviewer: fair, fair.

> If needed, the application would heed to request a new screen wake lock (the current `WakeLockSentinel` cannot be reused).

Reviewer: ```suggestion
If a subsequent screen wake lock is required, the application will need to request a new screen wake lock (the current `WakeLockSentinel` cannot be reused).
```

## Value

> A boolean value, that is `false` until the {{domxref("WakeLockSentinel")}} has been released (either through a call to {{domxref("WakeLockSentinel.release()")}} or because the lock has been released automatically) and the {{domxref("WakeLockSentinel/release_event", "release")}} event has been emitted, after which it becomes `true` and no longer changes.

Reviewer: ```suggestion
A boolean value that is `false` until the {{domxref("WakeLockSentinel")}} has been released (either through a call to {{domxref("WakeLockSentinel.release()")}} or because the lock has been released automatically) and the {{domxref("WakeLockSentinel/release_event", "release")}} event has been emitted, after which it becomes `true` and no longer changes.
```

## Examples

> This example shows how **`released`**'s value changes within a {{domxref("WakeLockSentinel")}}'s life cycle.

Reviewer: ```suggestion
This example shows how the value of `released` changes within a {{domxref("WakeLockSentinel")}}'s life cycle.
```
